### DISCUSSION TITLE
Downloading a huge document encodes the JSON as a string

### DISCUSSION THREAD

**Discussion by krivulcik** [ravendb/ravendb#16145] [Q&A] [answered]
The interface shown when opening a huge document in Studio offers a button to "Download document". This document is then downloaded as an encoded JSON string, e.g.:

```
"{\n    \"Name\": \"d1\",\n    \"HugeText\": \"lorem ipsum dolor sit amet consectetur adipiscing
...
metus interdum \",\n    \"@metadata\": {\n        \"@collection\": \"HugeDocuments\",\n        \"Raven-Clr-Type\": \"RavenDB.HugeDocument, RavenDB.Tests\"\n    }\n}"
```

An example is available here right now: http://live-test.ravendb.net/studio/index.html#databases/edit?&database=test-huge-document&id=d1

**Comment by ml054** [comment, marked as answer]
@krivulcik 

Yes it looks like an issue. I've reported it here:
https://issues.hibernatingrhinos.com/issue/RavenDB-20239/Huge-document-is-download-as-text-instead-as-a-JSON

Thanks for letting us know. :)
